props/saber_fett263_buttons.h

```diff
@@ -3712,6 +3818,16 @@ SaberFett263Buttons() : PropBase() {}
           sound_library_.SayNumber(calc_, SAY_WHOLE);
 	}
         break;
+      case MENU_IGNITION_OPTION2:
+      case MENU_RETRACTION_OPTION2:
+        if (SaberBase::IsOn()) {
+          calc_ += (direction * 1000);
+          if (calc_ > 32768) calc_ = 0;
```

Comment:
just do:
calc_ &= 0x7fff;

This can replace this line and the next one.

Reply:
replaced.